portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //why doesnt the headers in html format work and only show up as chars?
```

Comment:
Shall we remove this comment from here?

Reply:
Yeah, I was wondering about that question, do you know why that may be the case that the html format didn't show up properly?